wordpress/README.md

```diff
@@ -8,7 +8,8 @@ To be added
 
 
 ## Online resources
-To be added
+1. [Understanding how WordPress works]-
+   https://themes.artbees.net/blog/how-wordpress-works
```

Comment:
I dont think this is correct implementation. It is showing up like this:
<img width="770" alt="image" src="https://user-images.githubusercontent.com/12053186/155841550-30e6a181-a133-4e89-97d7-96e9b7ba5132.png">

You can check yourself [at this here](https://github.com/ColoredCow/resources/tree/Adding-link/wordpress)

It should show up like this instead (the link should be hidden)
<img width="201" alt="image" src="https://user-images.githubusercontent.com/12053186/155841568-8ed358fd-fc48-438f-9a6c-178f0401a324.png">